subprojects/common/graph/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id("convention.kotlin-jvm")
+    id("convention.publish-kotlin-library")
+    id("convention.libraries")
+}
+
+dependencies {
+    implementation(libs.jdgraphtCore)
+}
```

Comment:
New module for graph algorithms.
For now, use it only as a glue for [JGraphT](https://jgrapht.org). I found it convenient and robust.